src/main/kotlin/no/nav/syfo/oppfolgingsplan/api/OppfolgingsplanApi.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.syfo.oppfolgingsplan
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+
+fun Route.registerOppfolgingsplanApi(
+    oppfolgingsplanService: OppfolgingsplanService,
+) {
+    delete("/oppfolgingsplan/slett/{id}") {
+        try {
+            val id = call.parameters["id"] ?: throw IllegalArgumentException()
+
+            call.respond(HttpStatusCode.OK, oppfolgingsplanService.deleteOppfolgingsplan(id))
+        } catch (e: Exception) {
+            call.respond(HttpStatusCode.NotFound, "Could not find oppfolgingsplan with id")
+        }
+    }
+
+    delete("/oppfolgingsplan/slett/person/{fnr}") {
+        try {
+            val fnr = call.parameters["fnr"] ?: throw IllegalArgumentException()
+
+            call.respond(HttpStatusCode.OK, oppfolgingsplanService.deleteOppfolgingsplanerForUser(fnr))
```

Comment:
Det byttes på å bruke person og User for samme objekt her, kan være greit å velge en? Evt være mer spesifikk på hvem det er (arbeidstaker feks?)

Reply:
Endret til arbeidstaker :)